modules/help_editor/test/help_editorTest.php

```diff
@@ -1,42 +1,233 @@
 <?php
 /**
- * help_editor automated integration tests
+ * Help_editor automated integration tests
  *
  * PHP Version 5
  *
  * @category Test
  * @package  Loris
- * @author   Ted Strauss <ted.strauss@mcgill.ca>
+ * @author   Wang Shen <wangshen.mcin@gmail.com>
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris
  */
 
-require_once __DIR__ . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
-class helpEditorTestIntegrationTest extends LorisIntegrationTest
+require_once __DIR__."/../../../test/integrationtest"
+."s/LorisIntegrationTest.class.inc";
+ /**
+  * Help_editor automated integration tests
+  *
+  * PHP Version 5
+  *
+  * @category Test
+  * @package  Loris
+  * @author   Wang Shen <wangshen.mcin@gmail.com>
+  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+  * @link     https://github.com/aces/Loris
+  */
+class HelpEditorTestIntegrationTest extends LorisIntegrationTest
 {
     /**
+     * Insert testing data into the database
+     *
+     * @return none
+     */
+    function setUp()
+    {
+        parent::setUp();
+        $window = new WebDriverWindow($this->webDriver);
+        $window->maximize();
+        $this->DB->insert(
+            "help",
+            array(
+             'helpID'   => '999999',
+             'parentID' => '-1',
+             'hash'     => '7292dd87f9a200f21bf40ded779a58c2',
```

Comment:
instead of having a magic value, can you write this as md5("the string being hashed") so that it's more obvious to people reading the test what this is?